specification/trace/semantic_conventions/database.md

```diff
@@ -364,4 +365,18 @@ Furthermore, `db.name` is not specified as there is no database name in Redis an
 | `db.cosmosdb.sub_status_code`        | `0` |
 | `db.cosmosdb.request_charge`         | `7.43` |
 
+### Elasticsearch
```

Comment:
Given there's a `elasticsearch.md` now, I would move the examples there instead of here. It will already help in the migration that @AlexanderWert in #144

Reply:
ok, I've moved it. Thanks!

Reply:
Perfect! thank you :)